### PR TITLE
5.1.4 English Fixes

### PR DESCRIPTION
**Pull request by gwillikers10**

### CODE REVIEW

File: lang/english.xml [hunk 1]
```diff
@@ -23,29 +23,29 @@
...
+			<Text>+4 [ICON_Strength] Combat Strength for reaching Renaissance Era (Roosevelt Corollary).</Text>
 		</Row>
 		<Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_INDUSTRIAL" Language="en_US">
-			<Text>+5 [ICON_Strength] Combat Strength for reached Industrial Era (Roosevelt Corollary).</Text>
+			<Text>+5 [ICON_Strength] Combat Strength for reaching Industrial Era (Roosevelt Corollary).</Text>
 		</Row>
 		<!-- = leader ability (Abraham Lincoln)= -->
 		<Replace Tag="LOC_TRAIT_LEADER_LINCOLN_DESCRIPTION" Language="en_US">
-			<Text>Builds Industrial Zone district in half the time. Industrial Zone give +2 [ICON_Amenities] Amenities. Receive a free Melee unit after constructing Industrial Zone and their buildings. The free unit does not require resources when created or to maintain and receives +5 [ICON_Strength] Combat Strength.</Text>
+			<Text>+100% production toward . Industrial Zone give +2 [ICON_Amenities] Amenities. Receive a free Melee unit after constructing Industrial Zone and their buildings. The free unit does not require resources when created or to maintain and receives +5 [ICON_Strength] Combat Strength.</Text>
```
Comment (ZhenjaMax) [inline lang/english.xml:45]: Production (capitalized)
towards Industrial Zone District
?

Comment (gwillikers10) [reply lang/english.xml:45]: Production was not capitalized in Eleanor's ability, but probably should be in both here and on Eleanor.

File: lang/english.xml [hunk 2]
```diff
@@ -849,11 +849,11 @@
...
-			<Text>+200 [ICON_Gold] Gold when you conquer a city for the first time, +100 [ICON_GOLD] Gold when you capture a [ICON_BARBARIAN] Barbarian Outpost, increasing to +500 [ICON_Gold] Gold for both with Steel technology (on Standard speed).[NEWLINE][NEWLINE]Receive a free melee unit each time you found a city. Capturing and retaining at least one city owned by other civilization and founded by any major civilization grants extra Wildcard policy slot in any [ICON_Government] government.</Text>
+			<Text>+200 [ICON_Gold] Gold when you conquer a city for the first time, +100 [ICON_GOLD] Gold when you capture a [ICON_BARBARIAN] Barbarian Outpost, increasing to +500 [ICON_Gold] Gold for both with Steel technology (on Standard speed).[NEWLINE][NEWLINE]Receive a free melee unit each time you found a city. Capturing and retaining at least one city owned by another major civilization and founded by any major civilization grants extra Wildcard policy slot in any [ICON_Government] government. Gains an extra Wildcard Policy slot when capturing and retaining at least one city founded by a major civilization and owned by a major civilization at the time of capture (limit 1 wildcard slot)</Text>
```
Comment (ZhenjaMax) [inline lang/english.xml:856]: Duplicated description?

**Comment by ZhenjaMax** [comment]
Please, close PR
Brought these changes to #89

**Pull request closed without merging**